service/controller/appcatalog/v1/resource/index/create.go

```diff
@@ -0,0 +1,53 @@
+package index
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/giantswarm/microerror"
+	corev1 "k8s.io/api/core/v1"
+)
+
+func (r *Resource) ApplyCreateChange(ctx context.Context, obj, createChange interface{}) error {
+	cm, err := toConfigMap(createChange)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+	if cm.Name != "" {
```

Comment:
Newlines are free. Let's use them to make reading code a bit easier. 

```suggestion

	if cm.Name != "" {
```